src/components/sbb-link/sbb-link.tsx

```diff
@@ -17,43 +17,56 @@ import { i18nTargetOpensInNewWindow } from '../../global/i18n';
 export class SbbLink {
   /**
    * If set to true, the browser will
-   * show the download dialog on click.
+   * show the download dialog on click (optional).
    */
   @Prop() public download?: boolean;
 
-  /** The href value you want to link to */
-  @Prop() public hrefValue!: string;
+  /** The href value you want to link to (mandatory otherwise the link becomes a button)*/
+  @Prop() public href!: string;
 
   /**
    * The icon name we want to use,
    * choose from the small icon variants from
    * the ui-icons category from here
-   * https://lyne.sbb.ch/tokens/icons/.
+   * https://lyne.sbb.ch/tokens/icons/ (optional).
    * Inline variant doesn't support icons.
    */
   @Prop() public icon?: string;
```

Comment:
At the moment there is a slot and a property for the icon. The property is only used to toggle showing the icon slot, so there is completely no matter what you write in the property. With the new sbb-icon component we can handle now the property and the slot. Please change the logic to use a sbb-icon inside the icon slot as a default and set the name attribute of it to the icon property. Whenever a consumer provides its own icon slot, it is overwritten and if not it takes the string from the icon property. Please also change the corresponding stories if necessary.